src/realm.js

```diff
@@ -272,10 +286,11 @@ export class Realm {
       result = [c, astGenerator, astBindings, astProperties, astCreatedObjects];
       if (c instanceof PossiblyNormalCompletion) {
         let savedEffects = context.savedEffects;
-        invariant(savedEffects !== undefined);
```

Comment:
Now that PossiblyNormalCompletions can propagate out of functions this invariant is no longer true.